src/DynamoPublishTests/Class1.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DynamoPublishTests
+{
+    public class Class1
```

Comment:
It's fine that this is empty for now, but let's leave a comment making it clear why it's empty for now.